### ISSUE TITLE
SYSCONFDIR not honored?

### ISSUE THREAD

**Issue by ConiKost** [dunst-project/dunst#1041]
### Issue description
If I do compile dunst with ```SYSCONFDIR=/etc```, it seems ignored by dunst.

Looking with strace, it still searches the wrong directory.
```
newfstatat(AT_FDCWD, "/home/conikost/.config/dunst/dunstrc", 0x7ffc72ff2650, 0) = -1 ENOENT (Datei oder Verzeichnis nicht gefunden)
newfstatat(AT_FDCWD, "/usr/local/share/dunst/dunstrc", 0x7ffc72ff2650, 0) = -1 ENOENT (Datei oder Verzeichnis nicht gefunden)
newfstatat(AT_FDCWD, "/usr/share/dunst/dunstrc", 0x7ffc72ff2650, 0) = -1 ENOENT (Datei oder Verzeichnis nicht gefunden)
```

But according to build log, it being clearly passed with ```-DSYSCONFDIR```.

```
make -j5 -l4 SYSCONFDIR=/etc SYSTEMD=0 WAYLAND=1
[..]
x86_64-pc-linux-gnu-gcc -o src/dunst.o -c src/dunst.c -Wno-gnu-zero-variadic-macro-arguments -D_DEFAULT_SOURCE -DVERSION=\""1.8.0 (2022-02-24)"\" -DSYSCONFDIR=\"/etc\"  -g -std=gnu99 -pedantic -Wall -Wno-overlength-strings  -DENABLE_WAYLAND  -march=native -mfpmath=sse,387 -O3 -frecord-gcc-switches -fomit-frame-pointer -malign-data=abi -mtls-dialect=gnu2 -pipe -I/usr/include/glib-2.0 -I/usr/lib64/glib-2.0/include -I/usr/lib64/libffi/include -I/usr/include/libmount -I/usr/include/blkid -I/usr/include/gdk-pixbuf-2.0 -I/usr/include/libpng16 -I/usr/include/pango-1.0 -I/usr/include/harfbuzz -I/usr/include/freetype2 -I/usr/include/fribidi -I/usr/include/uuid -I/usr/include/cairo -I/usr/include/pixman-1 -pthread  -MMD -MP
[..]
```

Any ideas, why dunst is not using ```/etc/dunst``` as path for global config lookup?

### Installation info

- Version: 1.8.0
- Install type: manually
- Window manager / Desktop environment: i3

**Comment by fwsmit** [comment]
Thank you for reporting this issue. It should be fixed by #1042.

**Comment by ConiKost** [comment]
Ack, thank you!
